api/api.proto

```diff
@@ -1159,31 +1176,31 @@ service OpenStorageCloudBackup {
   rpc Enumerate(SdkCloudBackupEnumerateRequest)
     returns (SdkCloudBackupEnumerateResponse) {
       option(google.api.http) = {
-        get: "/v1/cloudbackups"
+        get: "/v1/cloudbackups/volume"
       };
     }
 
   // Status returns the status of any cloud backups of a volume
   rpc Status(SdkCloudBackupStatusRequest)
     returns (SdkCloudBackupStatusResponse) {
       option(google.api.http) = {
-        get: "/v1/cloudbackups/volume/{volume_id}/status"
+        get: "/v1/cloudbackups/volume/status"
```

Comment:
I didn't get part of avoiding volume_id here, get request will be ```"/v1/cloudbackups/volume/status``` for which volumes or all of them